src/rotate.c

```diff
@@ -0,0 +1,38 @@
+#include "rotate.h"
+
+void rotate_block(Block* b, const int rotation_matrix[2][2]){
+    /* Rotates block 90 degrees
+    using matrix multiplication with rotation matrix.*/
+    int matC[4][2]={0};
+    int i,j,k,z,r=4,c=2;
```

Comment:
`z` is unused here, and i don't think you need to declare any of these ints. you can do `for (int i...)` and compile with make